### ISSUE TITLE
Horizontal Review: offer review opportunity to TAG

### ISSUE THREAD

**Issue by burnburn** [w3c/did-core#119]
The chairs need to offer the Technical Architecture Group an opportunity to review if they wish, making clear that our group does not consider their input to be blocking for progress.  In other words, our document will continue to progress whether we receive their input or not, but they are encouraged to give us any feedback they may have earlier in the process rather than later.

**Comment by burnburn** [comment]
We will start this up when the document text is a little more coherent.

**Comment by iherman** [comment]
Discussed on 2020-04-14: when the current §3 and §6 gets filled in (which means that the final structure of the document is in place), we can go out to get the horizontal review done.

**Comment by brentzundel** [comment]
Related to Issue #94

**Comment by burnburn** [comment]
When ready, we need to create an issue [here](https://github.com/w3ctag/design-reviews/issues/new/choose) that will begin the TAG review.  Note that we should have completed the PING security review already and know whether we want them to respond with one issue in our repo for each point, one issue for all of their points, or to leave feedback as a comment to this new issue in their repo.

**Comment by burnburn** [comment]
Chairs will start this when #92 and #291 are done.

**Comment by brentzundel** [comment]
Chairs have reached out to the TAG and asked them to review the spec.

**Comment by msporny** [comment]
We had a preliminary discussion with TAG. We offered to meet with them to go over the specification. TAG is waiting on security/privacy questionnaire from us.

**Comment by brentzundel** [comment]
[The TAG issue](https://github.com/w3ctag/design-reviews/issues/556#issuecomment-746868177) we opened to request review from them has been updated with a link to the Security and Privacy Questionnaire

**Comment by rhiaro** [comment]
TAG is meeting today (right after WG meeting) specifically about this, so you should hear more soon.

**Comment by brentzundel** [comment]
TAG has indicated their review is complete.

**Comment by brentzundel** [comment]
Horizontal review is complete, closing
